source/openapi3/split.yaml

```diff
@@ -4263,6 +4262,169 @@ components:
             metadata:
               customer_id: xur4492
               product_ref: TSXL392110x
+    TransactionResponse:
+      title: A transaction (response)
+      required:
+        - ref
+        - parent_ref
+        - type
+        - category
+        - created_at
+        - matured_at
+        - cleared_at
+        - bank_ref
+        - status
+        - status_changed_at
+        - party_contact_id
+        - party_name
+        - party_nickname
+        - party_bank_ref
+        - description
+        - amount
+        - bank_account_id
+        - channels
+        - current_channel
+      type: object
+      properties:
+        ref:
+          type: string
+          description: The ref of the transaction (`C.*` or `D.*`)
+        parent_ref:
+          type: string
+          description: The ref of the parent of this transaction
+        type:
+          type: string
+          description: The type of the transaction
+          enum:
+            - credit
+            - debit
+        category:
+          type: string
+          description: The category of the transaction
+          enum:
```

Comment:
To ensure it remains up-to-date, shall we add a link from relevant parts of the application to here?